packages/blockchain-data/sdk/package.json

```diff
@@ -34,7 +34,7 @@
   "repository": "immutable/ts-immutable-sdk.git",
   "scripts": {
     "build": "NODE_ENV=production rollup --config rollup.config.js",
-    "dev": "rollup --config rollup.config.js -w",
+    "d": "swc src -d dist --strip-leading-paths --ignore '**/*.test.*'",
```

Comment:
We can not keep using `dev` as the script name because `nx` treat `dev` script output as un-cacheable.